shim/dist/Makefile

```diff
@@ -1,6 +1,6 @@
 PROJECT_DIR := $(shell cd ../..; pwd)
 RPMBUILD_DIR := $(shell mktemp -u /tmp/rpmbuild.XXXX)
-RELEASE_TARBALL_URL := https://github.com/alibaba/inclavare-containers/archive/v$(INCLAVARE_CONTAINERS_VERSION).tar.gz
+RELEASE_TARBALL_URL := https://github.com/inclavare-containers/archive/v$(INCLAVARE_CONTAINERS_VERSION).tar.gz
```

Comment:
`https://github.com/inclavare-containers/archive/v%{version}tar.gz` is not a valid url.